procurement_service/models/procurement_order.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models, api
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    @api.multi
+    def _is_procurement_service(self):
+        return self.product_id._is_service_buy_make_to_order() or False
```

Comment:
Why is this method needed? `self.product_id._is_service_buy_make_to_order()` already gives the desired boolean value.

Reply:
It is compliant with the module as it was in 8.0, so it doesn't break inheriting modules.
Moreover, this allows overriding the behaviour for the order without the necessity to override product's behaviour.

Reply:
Could you point me some module which depends on this method? I'm not capable to figure out an use case. Thanks!

Reply:
I don't know any module inheriting from this, but since this method already existed in v8 it is possible that some module exists, I prefer to keep this method for the reasons I said above.
Isn't it preferrable to keep inheritance intact as long as it is possible?

Reply:
Well, this is a module from a repo outside the OCA and 2 versions back, now you are proposing it to OCA so we have the chance to refactor as needed to make it useful and generic enough to make it valid for many usecases. Plus, no one has ported the module (not even to 9) and you (the one porting it now) don't know of any module depending on it... the chances are that no one is using it in v10.

summarizing :sweat_smile: : now is the moment to discuss about how to properly implement the module

Reply:
Looking at things from that perspective yes, this method is useful just because it is a shortcut.
I have found that it is used [here](https://github.com/OCA/manufacture/pull/232/files#diff-2c2d8c588716d0c2dabeaae23db8f46eR14) (almost forgot about it), but since that is my PR too I can change them both.

Reply:
ok, if you find it useful let's keep it.